### DISCUSSION TITLE
Detection phase output

### DISCUSSION THREAD

**Discussion by bbkgh** [buildpacks/community#124] [Q&A]
Hi. Is there any method/command for just detecting applicable buildpacks over a source code directory? i want to just detect type of source code as fast as possible (without actually building image).
thanks

**Comment by natalieparellano** [comment]
A bit late here, but in case others also have this question - this isn't possible with pack, but you can invoke the lifecycle directly from a builder to see the detect output. Something like the below command would work:

```
docker run -v $PWD:/workspace --entrypoint /cnb/lifecycle/detector cnbs/sample-builder:bionic -log-level debug
```
